src/Infrastructure/Persistence/ApplicationDbContext.cs

```diff
@@ -64,6 +65,10 @@ protected override void OnModelCreating(ModelBuilder builder)
         {
             builder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());
 
+            builder.ApplyConfiguration(new TodoListConfiguration());
```

Comment:
Not necessary to apply configurations individually, since they will be applied automatically with:
            builder.ApplyConfigurationsFromAssembly(Assembly.GetExecutingAssembly());